pages/events.js

```diff
@@ -1,8 +1,74 @@
+import Image from "next/image";
+import { Col, Container, Row, Button } from "react-bootstrap";
+import EventsData from '../assets/data/events.json';
+
+function Current(props) {
+  const {src, title, time, desc, rsvp_src} = props;
+  
+  return (
+    <Container className="pt-5 pb-5 mt-5 mb-5">
+      <Row className="align-items-center">
+        <Col lg={6}>
+          <Image
+            className="rounded"
+            objectFit={'contain'}
+            src={src}
+            alt={`${title} image`}
+            height={400}
+            width={400}
+          />
+        </Col>
+        <Col lg={6}>
+          <Container>
+            <h1 className="mb-4">{title}</h1>
+            <h5 className="mb-4">{time}</h5>
+            <p>{desc}</p>
+            <Button size="lg" href={rsvp_src}>RSVP</Button>
+          </Container>
+        </Col>
+      </Row>
+    </Container>
```

Comment:
To address your comment about code duplication, you could pass in a prop (maybe `isCurrent`) to indicate whether or not the event is current. Then you can conditionally render the rsvp button.

```suggestion
function Event(props) {
  const {src, title, time, desc, isCurrent, rsvp_src} = props;
  
  return (
    <Container className="pt-5 pb-5 mt-5 mb-5">
      <Row className="align-items-center">
        <Col lg={6}>
          <Image
            className="rounded"
            objectFit={'contain'}
            src={src}
            alt={`${title} image`}
            height={400}
            width={400}
          />
        </Col>
        <Col lg={6}>
          <Container>
            <h1 className="mb-4">{title}</h1>
            <h5 className="mb-4">{time}</h5>
            <p>{desc}</p>
            {isCurrent && 
                <Button size="lg" href={rsvp_src}>RSVP</Button>
            }
          </Container>
        </Col>
      </Row>
    </Container>
```

Reply:
Probably other ways to go about it - this is just one suggestion ¯\_(ツ)_/¯